internal/buildengine/engine.go

```diff
@@ -905,6 +905,10 @@ func (e *Engine) buildWithCallback(ctx context.Context, callback buildCallback,
 				return err
 			}
 		}
+		err = WriteGenericSchemaFiles(newSchemas, metasMap)
```

Comment:
So this approach will write out the generated schema after the build, which then results in clients being generated?

Won't this also generate new copies of things like Data classes that will then be duplicated?

I guess with this approach you have to check the schema in, as projects won't be buildable otherwise.

Reply:
This doesn't change the approach much. Previously it wrote schema files inside the generated stubs function on line 903. Stubs are only written for modules _not_ in the last topology group. ie modules that have dependant modules.

Now it does the same but outside the if statement, for every group in the topology.